pkg/controller/jobs/jobset/jobset_webhook.go

```diff
@@ -58,6 +59,20 @@ func (w *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	jobSet := fromObject(obj)
 	log := ctrl.LoggerFrom(ctx).WithName("jobset-webhook")
 	log.V(5).Info("Applying defaults", "jobset", klog.KObj(jobSet))
+
+	if wlName, hasPrebuilt := jobframework.PrebuiltWorkload(jobSet); hasPrebuilt {
```

Comment:
Can you explain why this is needed? Also in a comment. I'm puzzled because we only support parent-workload annotation for Job at the moment. If this is a drive-by fix, or feature please move it to a dedicated PR.

Reply:
> I'm puzzled because we only support parent-workload annotation for Job at the moment.

Jobs is the user of the annotation, however Jobsets  are one of the potential owners of such Jobs.

> If this is a drive-by fix, or feature please move it to a dedicated PR.

It's not.

Reply:
ok, I see.